CMakeLists.txt

```diff
@@ -29,7 +31,14 @@ message(STATUS "STANDALONE_BUILD [${STANDALONE_BUILD}] STANDALONE_BUILD_SHARED [
 function ( clone_open62541 )
   SET (OPEN62541_VERSION "0.2")
   message(STATUS "cloning open62541 from github. *NOTE* cloning tag [${OPEN62541_VERSION}]")
```

Comment:
small point but the message "cloning ..." is perhaps not wholly accurate anymore. Perhaps we should change the verb "cloning" to "using/fetching" since we delegate to FetchContent_*()

Infact, just to keep naming representative, perhaps we shoudl rename function 
```clone_open62541```
to 
```fetch_open62541```